generate-release/src/main.rs

```diff
@@ -55,10 +55,18 @@ enum Commands {
         #[arg(short, long)]
         overwrite_existing: bool,
     },
+    /// Generates release notes for all PRs merged with the `C-Needs-Release-Note` label.
+    ///
+    /// This will also open an issue for each PR that doesn't have a release note.
+    /// While duplicate issues will not be opened, be sure to use the `dry-run` flag
```

Comment:
```suggestion
    /// While duplicate issues will not be opened, be sure to use the `--dry-run` flag
```

Adding the double-dash makes it more clear that this flag is supposed to be used in the CLI, not somewhere else. This is a stylistic choice, though, and I won't block on it. :)